src/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -57,19 +58,30 @@ public AcquireTokenForClientParameterBuilder(IConfidentialClientApplication conf
         }
 
         /// <summary>
+        /// Specifies if the confidential client application should force refreshing the
```

Comment:
This is a bit confusing. I think you mean to say "force acquiring a new access token based on the refresh token, instead of just acquiring the access token from the cache" ?

Reply:
I took Bogdan's suggestion, but am leaving as open in case you disagree.

---
In reply to: [248610789](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/793#discussion_r248610789) [](ancestors = 248610789)

Reply:
#Solved